sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appservice/samples/ManageLinuxWebAppCosmosDbByMsi.java

```diff
@@ -167,13 +167,18 @@ public static boolean runSample(AzureResourceManager azure, String clientId) thr
             DockerClient dockerClient = DockerUtils.createDockerClient(azure, rgName, region,
                     azureRegistry.loginServerUrl(), acrCredentials.username(), acrCredentials.accessKeys().get(AccessKeyType.PRIMARY));
 
+            String imageName = "tomcat:7.0-slim";
             String privateRepoUrl = azureRegistry.loginServerUrl() + "/todoapp";
-            dockerClient.buildImageCmd(new File(ManageLinuxWebAppCosmosDbByMsi.class.getResource("/todoapp-cosmosdb/Dockerfile").getFile()))
```

Comment:
Long term, we might probably try fix this docker file (or rather, use JAR deployment directly).

Reply:
I think maybe JAR is better, corresponding to windows version.